src/library/trackmodel.h

```diff
@@ -15,13 +16,15 @@ class TrackModel {
   public:
     static const int kHeaderWidthRole = Qt::UserRole + 0;
     static const int kHeaderNameRole = Qt::UserRole + 1;
+    static const int positionChangeLimit = 10; // varuable that avoids to too fast scroll
 
     TrackModel(QSqlDatabase db,
                const char* settingsNamespace)
             : m_db(db),
               m_settingsNamespace(settingsNamespace),
               m_iDefaultSortColumn(-1),
-              m_eDefaultSortOrder(Qt::AscendingOrder) {
+              m_eDefaultSortOrder(Qt::AscendingOrder),
+              m_position(0) {
```

Comment:
Initalize order does not match the order of the definition in the header. 
You can either mover m_position up here or move the definition to the bottom of the class definition. 

This is the compiler warning: 
```
src/library/trackmodel.h: In constructor ‘TrackModel::TrackModel(QSqlDatabase, const char*)’:
src/library/trackmodel.h:174:19: warning: ‘TrackModel::m_eDefaultSortOrder’ will be initialized after [-Wreorder]
     Qt::SortOrder m_eDefaultSortOrder;
                   ^
src/library/trackmodel.h:168:9: warning:   ‘int TrackModel::m_position’ [-Wreorder]
     int m_position;
         ^
src/library/trackmodel.h:21:5: warning:   when initialized here [-Wreorder]
     TrackModel(QSqlDatabase db,

```